takahe/settings.py

```diff
@@ -299,10 +304,12 @@ class Config:
     elif parsed.scheme == "s3":
         DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
         AWS_STORAGE_BUCKET_NAME = parsed.path.lstrip("/")
```

Comment:
Not sure if in scope for this PR, but we can also extract a value for `AWS_LOCATION` from the path with `split("/", 1)` .

Something along the lines of:
```
s3_parts = parsed.path.lstrip("/").split("/", 1)
AWS_STORAGE_BUCKET_NAME = s3_parts[0]
if len(s3_parts) == 2:
    AWS_LOCATION = s3_parts[1]
```

Reply:
Probably out of scope until we figure out path storage in a shared bucket?

Or, more possibly, pushing images through a CDN front.